### PR TITLE
Policy prefix common field

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**
https://github.com/gravitee-io/issues/issues/6614

**Description**

Prefix description and condition with `common`
As we make a single form for the policy settings, the prefix allows to differentiate description and conditional from the policy's own fields.
📝 The conditional key is already defined in some policies for other aspects

**Additional context**

### CODE REVIEW

**Review by ytvnr — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A beta version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-components@3.25.1-policy-prefix-common-field-0a1bbd9
```

File: src/policy-studio/gv-policy-studio.js [hunk 1]
```diff
@@ -841,18 +841,18 @@ export class GvPolicyStudio extends KeyboardElement(LitElement) {
   }
 
   _writeFlowStep(values) {
-    const { description, condition, ...configuration } = values;
+    const { commonDescription, commonCondition, ...configuration } = values;
     if (
       this._currentFlowStep.step._new ||
-      this._currentFlowStep.step.description !== description ||
-      this._currentFlowStep.step.condition !== condition ||
+      this._currentFlowStep.step.commonDescription !== commonDescription ||
```
Comment (ThibaudAV) [inline src/policy-studio/gv-policy-studio.js:847]: `this._currentFlowStep.step.description !== commonDescription ||`

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A beta version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-components@3.25.1-policy-prefix-common-field-6fb7e3e
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A beta version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-components@3.25.2-policy-prefix-common-field-950eef3
```

**Pull request merged**